### PR TITLE
second step

### PR DESCRIPTION
**Pull request by Alex-zzzz**

### CODE REVIEW

**Review by Ivan95kos — CHANGES_REQUESTED** [review]
please, fix all checks
Знімок екрана 2023-10-24 о 18 53 35

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,11 +16,5 @@ jobs:
...
-      - uses: mate-academy/auto-approve-action@v2
-        if: ${{ github.event.pull_request && success() }}
-        with:
-          github-token: ${{ github.token }}
-      - uses: mate-academy/auto-reject-action@v2
-        if: ${{ github.event.pull_request && failure() }}
-        with:
-          github-token: ${{ github.token }}
\ No newline at end of file
+
```
Comment (kozhukhovsky) [inline .github/workflows/ci.yml]:
```suggestion
```

File: src/main/java/online/book/store/dto/BookDto.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package online.book.store.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookDto {
+
```
Comment (kozhukhovsky) [inline src/main/java/online/book/store/dto/BookDto.java]:
```suggestion
```

File: src/main/java/online/book/store/mapper/BookMapperImpl.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package online.book.store.mapper;
+
+import online.book.store.dto.BookDto;
+import online.book.store.dto.CreateBookRequestDto;
+import online.book.store.model.Book;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BookMapperImpl implements BookMapper {
+    private BookDto dto = new BookDto();
```
Comment (kozhukhovsky) [inline src/main/java/online/book/store/mapper/BookMapperImpl.java]: why is not local?

**Review by kozhukhovsky — APPROVED** [review]

**Pull request merged**